### ISSUE TITLE
Bump/loosen protobuf ping

### ISSUE THREAD

**Issue by frenck** [jfroy/aiobafi6#5]
For the Home Assistant project, we are currently working towards Python 3.11 support; some of our key packages, therefore, need an upgrade, including `protobuf`.

Home Assistant depends on `aiobafi6`, which pins `protobuf`:

```
aiobafi6==0.7.2
  - protobuf [required: >=3.20,<3.21, installed: 3.20.1]
  - zeroconf [required: >=0.38, installed: 0.39.4]
    - async-timeout [required: >=4.0.1, installed: 4.0.2]
    - ifaddr [required: >=0.1.7, installed: 0.1.7]
```

Source: https://github.com/jfroy/aiobafi6/blob/680976b34f41c2e06c6120a81271050dc13f3556/pyproject.toml#L19

However, we need to achieve `protobuf>=4.21.3` @ the Home Assistant end. Could the dependency requirement on `aiobafi6` be looser? Preferably, this package should pin it as, for example, `protobuf>=3.20.0`.

**Comment by frenck** [comment]
Fixed in cf77aca42c58fd0b3f24f23671438758b1076d8f
and released with the https://github.com/jfroy/aiobafi6/releases/tag/0.7.3 release

**Comment by jfroy** [comment]
Done in 0.7.3.
